.github/workflows/continuous-integration-workflow.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: maybe_disable_death_tests
         if: ${{ matrix.distro == 'fedora:rawhide' }}
         run: echo "GTEST_FILTER=-*DeathTest*" >> $GITHUB_ENV
+      - name: test_external_gtest
```

Comment:
FIXME: remove this and find a better way for testing (changing the `ci-containers`?)